handlers/done_task_handler.go

```diff
@@ -0,0 +1,72 @@
+package handlers
+
+import (
+	"database/sql"
+	"encoding/json"
+	"net/http"
+	"strconv"
+	"time"
+
+	"go_final_project/db"
+	"go_final_project/nextdate"
+)
+
+// Отметка о выполнении задачи
+func PostTaskDoneHandler(w http.ResponseWriter, r *http.Request) {
+	idStr := r.URL.Query().Get("id")
+
+	if idStr == "" {
+		respErr, _ := json.Marshal(map[string]string{"error": "id is empty"})
+		http.Error(w, string(respErr), http.StatusBadRequest)
+		return
+	}
+
+	idInt, err := strconv.ParseInt(idStr, 10, 64)
+	if err != nil {
+		respErr, _ := json.Marshal(map[string]string{"error": "id format is wrong"})
+		http.Error(w, string(respErr), http.StatusBadRequest)
+		return
+	}
+
+	// Проверка существования задачи (получаем задачу)
+	task, err := db.GetTask(idInt)
+	if err != nil {
+		if err == sql.ErrNoRows {
```

Comment:
Если обернуть ошибку, то так уже не получится, нужен такой подход
```suggestion
		if errors.Is(err, sql.ErrNoRows) {
```